sdk/python/featurestore_sample/setup-resources.sh

```diff
@@ -8,18 +8,23 @@ AML_WORKSPACE_NAME=$(az configure -l --query "[?name=='workspace'].value" -o tsv
 OUTPUT_COMMAND="print"
 FEATURE_STORAGE_ACCOUNT_NAME=${RESOURCE_GROUP}fs
```

Comment:
For my education, which version of control plane and data plane sdk are used for test? where it is defined?